### DISCUSSION TITLE
How to model a biomarker in MoBi

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#855] [Questions & Problems] [answered]
Dear all,

I am trying to model a biomarker in MoBi, so basically I would like to add a molecule and then define a baseline value, before beginning the modelling of the time course of the biomarker.
As a starting point, I used the option "Add PK-Sim molecule" in MoBi, and here I have a first question: When doing so, the molecule which is created, will it be considered a small or large molecule? In my case, I would like to have a floating (non-stationary) large molecule.

After I created my molecule, I tried to define the baseline value (e.g. 50µmol/l), but in the molecule BB there is only the option to provide an amount in µmol. I also tried to use the _Molecule start values_ but also there I could only find amounts. 
(From what I have seen, the problem in using the amounts, is that the system is not equilateral, so the simulation need a certain point to reach an equilibrium, while if I would set the same concentration in all compartments I should not have this issue).
I had the idea to use a formula as start amount of my biomarker, since in this case I could define something like:
_start_amount=50*volume_
By doing so, each organ, independently from the volume of the organ, should have my 50µmol/l. 
After I created this formula in the compound BB, nothing happened. I suspect that I need to connect this formula to the _Molecule start values_ but I did not manage to find a way. Could someone help with this?
In case I was not clear enough, let me know and I can try to clarify my question.

Thanks a lot in advance!

Donato

**Comment by HenrikCordes** [comment]
Hi @teutonicod,

I'm not sure if I completely understood your question but let’s discuss if not.

Once I did a biomarker the following way. 
I knew the clearance processes of my BM (renal & metabolic cl) 
I set up the BM as compound in PK-Sim together with an "user defined" continuous infusion from t 0 to t 1000 in the plasma compartment.
Than I created artificial experimental values for the BM in plasma, let's say 50 µmol/l at 200, 250, 300 & 400 h
With that setting you can use the Parameter Estimation in PK to optimize infusion dose / rate, optional compound parameters and if not known the magnitude of the cl processes.
As a result you will have a steady-state BM model with a baseline in every compartment. 

Than, I added my Drug of interest and the anticipated interaction (DDI) on the BM clearance enzyme with a dosing of the drug at 400 h (shift all experimental data points of the drug by +400h).

May this is of help. 

Best,
Henrik

**Comment by teutonicod** [comment]
Hi @HenrikCordes,

Thanks a lot for your reply, and I think you got exactly the point of my question! Indeed, this would be a work around, but in this case you will need to go to PK-Sim every time you would need to make changes in your biomarker administration. You will also need some time to reach the equilibrium in the system, but yes this would definitely be a way to do it!
Since I am already working with MoBi I would like to understand how to do it in MoBi directly, since I have seen some similar approach in some MoBi model (for instance the initial concentration of FcRn receptor I think is set in a similar way). This is also an opportunity for me to better understand how MoBi works :-)
Does anybody have any suggestion on how to do this in MoBi?

**Comment by HenrikCordes** [comment]
Hi @teutonicod,

if you want more than a quick-and-dirty solution in PK-Sim (which I usually prefer :-D ) I would like to point you to the work of Vanessa Baier, a former and talented co-worker of mine. She developed a more sophisticated way to do it in MoBi ;-)
In one of her previous work:  [physiology-based bile acid model (PBBM),](https://www.frontiersin.org/articles/10.3389/fphys.2019.01192) she included a plasma bile acid base line.

In the supplementary material of the linked publication you can find a step-by-step manual of how to _**"Extend PBPK model of G-CDCA in MoBi"**_. Here it is explained how you can estimate the steady-state of a BM (Bile acid) and set this as starting values for any other simulation. 

Maybe this helps.

Best,
Henrik

**Comment by teutonicod** [comment]
Thanks @HenrikCordes for the publication (very interesting by the way) and for your replies, but unfortunately it did not solve my problem.
Probably it will be easier with some more details.

I created the molecule in MoBi using the option "Add PK-Sim molecule". After this I set the start values of the molecule in the molecule BB as a formula, as explained in my first post. Below is the screenshot of how I did it

![image](https://user-images.githubusercontent.com/11489621/102768880-21fd7400-4382-11eb-8ceb-5cb756e77338.png)


Also by doing so, I think that MoBi is not actually using the formula to initiate the concentration in each compartment, since I see that by using this formula has no impact vs using a fixd value equal to 0. 
My explanation is that also if I set a formula as starting value in the molecule BB, MoBi still use the amounts reported in the _Molecule start values_ BB. So I think (please let me know if I am wrong), that in order to define a start amount for my molecule in the different compartments, I would need to have a formula in the _Molecule start values_ BB.
Now when I go there and try to use the formula I just created, I cannot find it in the scroll down menu on the right (See screenshot below). I also could not find a way to create a formula directly in the _Molecule start values_ BB.

![image](https://user-images.githubusercontent.com/11489621/102770159-fa0f1000-4383-11eb-9289-a7b01f487c68.png)


So to recap, my questions are:

- did I got it right? In order o use a start amount in each compartment defined by a formula, do I need to set up a formula in the _Molecule start values_ BB

- If so, how can I create a formula there?

- If the start amount of a molecule is defined by the _Molecule start values_ BB, why the user has the option to define a formula in the Molecule BB, if then this formula is not used anyway?

- When creating a molecule using the option "Add PK-Sim molecule" in MoBi, will the molecule parameterized as small or large molecule?

Hope that this clarifies what I would like to do, and my questions.

Thanks a lot!

Best,

Donato

**Comment by PavelBal** [comment, marked as answer]
Hi Donato.

>  In order o use a start amount in each compartment defined by a formula, do I need to set up a formula in the Molecule start values BB

Correct, the "Start values"-building blocks always win. In order to use the formula defined in the molecules-BB, select all entries for this molecule in the MoleculeStartValues (MSV)-BB and select "Start values -> Refresh selected from source"

![image](https://user-images.githubusercontent.com/25903040/102773761-0f873880-438a-11eb-97da-e4f5965e2042.png)

The values will be set to the formula defined in the Molecules-BB.

> If so, how can I create a formula there?

It is also possible to create new formulas in the MSV-BB by clicking on the "+"-sign in the "Formula"-field of the entry. It appears only when you click on this field.

>Now when I go there and try to use the formula I just created, I cannot find it in the scroll down menu on the right (See screenshot below).

Formulas are not shared between the BBs. So The MSV-BB has no access to the formulas created in the Molecules-BB.

> When creating a molecule using the option "Add PK-Sim molecule" in MoBi, will the molecule parameterized as small or large molecule?

There are no general differences in the parametrization between small and large molecules apart from the parameter `Is small molecule` , which is 0 by default. However, changing this value to 1 is not sufficient. Large molecules have additional "molecules" like the molecule-FcRn complex and reactions describing the binding to FcRn.

**Comment by teutonicod** [comment]
Hi @PavelBal,

Thanks a lot for your answers, indeed ti worked. About the molecule, if I am working in MoBi with a protein model (so I have all the additional components, such as FcRn in the model), by setting the `Is small molecule` to 0, the molecule should be treated as a large molecule in the model, right? Since I do not see any impact on the PK profile of the molecule when changing the Kd FcRn.
